.evergreen/config.yml

```diff
@@ -44,6 +44,7 @@ functions:
 
   bootstrap-mongo-orchestration:
     - command: subprocess.exec
+      type: test
```

Comment:
These are evergreen validation fixes related to errors produced on line [96](https://github.com/mongodb/mongo-hibernate/pull/124/files#diff-2bc841e86ce96b7b422ae203fd8315d0b2a461956cecbe0e096420656fc3fb12L96) and on line 110 & 124